lms/static/scripts/frontend_apps/utils/dashboard-opening-form-controller.tsx

```diff
@@ -0,0 +1,48 @@
+import { render } from 'preact';
+
+import type { DashboardConfig } from '../config';
+
+/**
+ * Handles the rendering and submitting of the form which is used to open the
+ * instructor dashboard
+ */
+export class DashboardOpeningFormController {
```

Comment:
Was there a particular reason for making this a class, as opposed to just a function?

Reply:
I think I had an idea when I started implementing it, around trying to make this in a similar way to other pieces of code that render small pieces of markup, but it ended up being pretty simple.

Considering it only has a public method, it could probably just be a function, yes.